README.md

```diff
@@ -48,11 +48,14 @@ You can use your favorite [ingress controller](https://kubernetes.io/docs/concep
 
 This chart is tested, maintained and shipped with default values for the nginx ingress controller.
 
-You can find documentation for nginx ingress controller [here](https://kubernetes.github.io/ingress-nginx/).
-Basic configuration options are detailled [here](https://kubernetes.github.io/ingress-nginx/deploy/) and you can find advanced configuration options [here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/).
+You can find documentation for nginx ingress controller [here](https://kubernetes.github.io/ingress-nginx/). Basic configuration options are detailled [here](https://kubernetes.github.io/ingress-nginx/deploy/) and you can find advanced configuration options [here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/).
```

Comment:
It seems we lost a new line between the 2 sentences.
Is it intentional or not ?